src/borg/testsuite/hashindex.py

```diff
@@ -125,6 +125,31 @@ def test_chunkindex_summarize(self):
         assert unique_chunks == 3
 
 
+class HashIndexExtraTestCase(BaseTestCase):
+    """These tests are separate because they should not become part of the selftest
+    """
+    def test_chunk_indexer(self):
+        # see _hashindex.c hash_sizes, we want to be close to the max fill rate
+        # because interesting errors happen there
+        max_key = int(65537 * 0.75) - 10
```

Comment:
How can I do that? As far as i know it's a C macro, and there is (still) no way to reach it.